utils/profile-services.ts

```diff
@@ -0,0 +1,217 @@
+import { LimitedProfile } from "@pagopa/io-functions-commons/dist/generated/definitions/LimitedProfile";
+import { RetrievedProfile } from "@pagopa/io-functions-commons/dist/src/models/profile";
+import { ServiceId } from "@pagopa/io-functions-commons/dist/generated/definitions/ServiceId";
+import {
+  RetrievedServicePreference,
+  ServicesPreferencesModel
+} from "@pagopa/io-functions-commons/dist/src/models/service_preference";
+import { asyncIterableToArray } from "@pagopa/io-functions-commons/dist/src/utils/async";
+import {
+  IResponseErrorNotFound,
+  ResponseErrorFromValidationErrors,
+  ResponseErrorNotFound
+} from "italia-ts-commons/lib/responses";
+import * as e from "fp-ts/lib/Either";
+import * as te from "fp-ts/lib/TaskEither";
+import * as t from "fp-ts/lib/Task";
+import { ServicesPreferencesModeEnum } from "@pagopa/io-functions-commons/dist/generated/definitions/ServicesPreferencesMode";
+import * as a from "fp-ts/lib/Array";
+import { IResponseErrorValidation } from "@pagopa/ts-commons/lib/responses";
+import { BlockedInboxOrChannelEnum } from "@pagopa/io-functions-commons/dist/generated/definitions/BlockedInboxOrChannel";
+import { TaskEither } from "fp-ts/lib/TaskEither";
+
+const NOT_FOUND_TITLE = "ServicesPreferences not found";
+
+interface IServicePreferenceHandler {
```

Comment:
We may want to have a comment explaining the rationale behind this interface

Reply:
I think you should report this here https://github.com/pagopa/io-functions-services/pull/136/files#diff-4e0957bac5b9ec17e7bb60a713b90d20a27ce51b0003541ac27fb476b911158b